gr-digital/include/gnuradio/digital/diff_stbc_decoder_cc.h

```diff
@@ -0,0 +1,87 @@
+/* -*- c++ -*- */
+/* 
+ * Copyright 2018 Free Software Foundation, Inc.
+ * 
+ * This file is part of GNU Radio
+ * 
+ * GNU Radio is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3, or (at your option)
+ * any later version.
+ * 
+ * GNU Radio is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with GNU Radio; see the file COPYING.  If not, write to
+ * the Free Software Foundation, Inc., 51 Franklin Street,
+ * Boston, MA 02110-1301, USA.
+ */
+
+
+#ifndef INCLUDED_DIGITAL_DIFF_STBC_DECODER_CC_H
+#define INCLUDED_DIGITAL_DIFF_STBC_DECODER_CC_H
+
+#include <gnuradio/digital/api.h>
+#include <gnuradio/block.h>
+
+namespace gr {
+  namespace digital {
+
+/*! \brief Decodes an incoming stream after the rules of a differential
+ * Space Time Block Code (STBC).
+ *
+ * The differential STBC decoder works with 'sequences' of length 2, decoding them
+ * into a code sequence of length 2.
+ * The number of input items is automatically scheduled to a multiple of 2, however,
+ * if the input data stream is terminated, the absolute number of input items
+ * must be an even number.
+ * When running with an infinite sequence stream, the differential STBC decoder is a sync
+ * block which produces the same amount of output items as there are input items.
+ * The code rate is R=1.
+ * If a finite block of sequences is transmitted, the decoder needs a reference sequence
+ * to differentially decode the first sequence. When using stream tags 'start'
+ * on the stream, the decoder therefore produces one sequence less than it consumes, because
+ * the tagged sequence is skipped and taken as reference to differentially decode the first
+ * sequence after the tag as first decoded output sequence. If there is no tag at the very
+ * first sequence of the transmission, the decoder uses a dummy sequence (= basis vectors of
+ * basis transformation 'd_basis_vecs') to decode the first sequence. Generally this leads
+ * to a wrong decoded first symbol. The following symbols are decoded correctly.
+ * To avoid a wrong decoded first symbol, there must be a stream tag on the first
+ * incoming sequence.
```

Comment:
You could mention what the stream tag has to look like to function correctly without requiring users to look into the code.